libguard/guard_interface.cpp

```diff
@@ -137,21 +137,31 @@ GuardRecord create(const EntityPath& entityPath, uint32_t eId, uint8_t eType)
     uint32_t offset = 0;
```

Comment:
commit message : "libguard: Changes in create api " please provide meaningfull subject. here looks ore generic one.

Reply:
Done.